esp/esdllib/esdl_transformer2.cpp

```diff
@@ -1111,8 +1111,12 @@ void Esdl2Struct::process(Esdl2TransformerContext &ctx, const char *out_name, Es
                                 if (ctx.writer->length() > len)
```

Comment:
Given dataForProcessed has been flagged since len set, is it possible for this condition to be false?